torch/utils/data/dataloader.py

```diff
@@ -261,7 +458,7 @@ def __init__(self, loader):
         if self.num_workers > 0:
             self.worker_init_fn = loader.worker_init_fn
             self.worker_queue_idx = 0
-            self.worker_result_queue = multiprocessing.Queue()
+            self.worker_result_queue = multiprocessing.JoinableQueue()
```

Comment:
There's no mention of why we use a `JoinableQueue` in the comment above.

Reply:
Only one reason: I forgot to change it back to Queue :P